ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ChapterNav.vue

```diff
@@ -32,6 +46,21 @@ export default {
     }
   },
 
+  watch: {
+    getCurrentCourse () {
```

Comment:
any reason for doing `watch` over just setting the selected campaign in `setDefaultCampaign`?

Reply:
The junior campaign is set as default somewhere else after `setDefaultCampaign` is ran. Using watch is necessary to switch the campaign after the campaign defaults to junior.

Reply:
any idea where it is set like that?